### ISSUE TITLE
Tokens not supported in querystring

### ISSUE THREAD

**Issue by fredrikjacobson** [slack-scala-client/slack-scala-client#184]
https://api.slack.com/changelog/2020-11-no-more-tokens-in-querystrings-for-newly-created-apps

It is no longer supported to send the token in the queryparam for token. 
`makeApiMethodRequest` still does this, leading to invalid_auth error for new apps.

**Comment by fredrikjacobson** [comment]
Works by replacing this

```
private val apiBaseWithTokenRequest = apiBaseRequest.withUri(
    apiBaseRequest.uri.withQuery(Uri.Query((apiBaseRequest.uri.query() :+ ("token" -> token)): _*))
  )

```

with this

```
private val apiBaseWithTokenRequest = apiBaseRequest.withHeaders(Authorization(OAuth2BearerToken(token)))
```

**Comment by fredrikjacobson** [comment]
https://github.com/slack-scala-client/slack-scala-client/pull/185

**Comment by mccartney** [comment]
closed by #185
